.github/ISSUE_TEMPLATE/linkcheck_report.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 🐛 Weekly linkcheck failed
+about: Use this template for checking broken links
+name: Weekly linkcheck failed
+labels: maintenance, bug
+---
+
+Links are checked weekly to make sure the documentation links areg working. Last check on {{ date | date('dddd, MMM Do YYYY, hh:mm A') }} failed. 
```

Comment:
```suggestion
Links are checked weekly to make sure the documentation links are working. Last check on {{ date | date('dddd, MMM Do YYYY, hh:mm A') }} failed. 
```
And is there documentation for the date field syntax in there? Just so I can check that too.